src/MainDashBoard/components/sideSheet/krCheckIn/KrCheckInInputs.tsx

```diff
@@ -0,0 +1,319 @@
+  const [isActiveBtn, setIsActiveBtn] = useState(false);
+
+  const handleTargetChange = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e.target.value === '') {
+      setTarget('');
+      return;
+    }
+    const rawValue = e.target.value.replace(/,/g, '').slice(0, 11);
+    if (/^\d*$/.test(rawValue)) {
+      const num = Number(rawValue).toLocaleString();
+      setTarget(num);
+    }
+  };
+
+  const handleLogContentChange = (e: ChangeEvent<HTMLTextAreaElement>) => {
+    if (e.target.value === '') {
+      setLogContentCount(0);
+      setLogContent('');
+    }
+    const lengthCount = limitMaxLength(e, 100);
+
+    if (!lengthCount) return;
+    setLogContentCount(lengthCount);
+    setLogContent(e.target.value);
+  };
+
+  useEffect(() => {
+    target && logContent ? setIsActiveBtn(true) : setIsActiveBtn(false);
+  }, [target, logContent]);
+  return (
+    <section css={enterLayoutStyles}>
+      <article css={enterArticleStyles}>
+        <span css={enterInputBoxStyles}>
+          <StLabel htmlFor="enterProgress">kr 수정</StLabel>
+          <StEditNum>
+            <span>통합 가입 수</span>
+            <StEditNumInput
+              id="enterProgress"
+              placeholder="200,000"
+              value={target}
+              onChange={handleTargetChange}
+            />
+            <span>건 돌파</span>
+          </StEditNum>
+        </span>
+        <span css={enterInputBoxStyles}>
+          <StLabel htmlFor="enterProgressCheckin">체크인</StLabel>
+          <CheckInInput
+            logContent={logContent}
+            handleLogContentChange={handleLogContentChange}
+            logContentCount={logContentCount}
+          />
+        </span>
+      </article>
+      <footer css={enterFooterStyles}>
+        <StCnclBtn onClick={onCancle}>취소</StCnclBtn>
+        <StEnterBtn1 isActiveBtn={isActiveBtn}>체크인 완료</StEnterBtn1>
+      </footer>
+    </section>
+  );
+};
+
+const enterLayoutStyles = css`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 100%;
+  height: 100%;
+  padding: 2rem 2.2rem 3.6rem;
+`;
+
+const enterArticleStyles = css`
+  display: flex;
+  flex-direction: column;
+  gap: 2.4rem;
+`;
+
+const enterFooterStyles = css`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin-top: 5rem;
+`;
+
+const StCnclBtn = styled.button`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 14.5rem;
+  height: 3rem;
+  color: ${({ theme }) => theme.colors.gray_000};
+  background-color: ${({ theme }) => theme.colors.gray_500};
+  border-radius: 6px;
+  ${({ theme }) => theme.fonts.btn_14_semibold};
+`;
+
+const StEnterBtn1 = styled(StCnclBtn)<{ isActiveBtn: boolean }>`
+  color: ${({ theme, isActiveBtn }) =>
+    isActiveBtn ? theme.colors.gray_600 : theme.colors.gray_000};
+  background-color: ${({ theme, isActiveBtn }) =>
+    isActiveBtn ? theme.colors.sub_mint : theme.colors.gray_500};
+`;
```

Comment:
```suggestion
const StEnterBtn1 = styled(StCnclBtn)<{ isActiveBtn: boolean }>`
  ${({ theme, isActiveBtn }) => css`
    color: ${isActiveBtn ? theme.colors.gray_600 : theme.colors.gray_000};
    background-color: ${isActiveBtn ? theme.colors.sub_mint : theme.colors.gray_500};
  `};
`;
```

같은 로직이니 이렇게 한 번에 묶어서 가능합니다